github/resources.py

```diff
@@ -0,0 +1,132 @@
+import functools
+import io
+import json
+import os
+
+import pyjq
+
+
+results_dir = os.environ["RESULTS_DIR"]
+organization = os.environ["Organization"]
+today = os.environ["TODAY"]
+org_list = organization.split()
+
+aux_files = {
+    "repos_of_interest": "metadata_repos.json"
+}
+
+
+class GitHubException(Exception):
+    pass
+
+
+class GitHubFileNotFoundException(GitHubException):
+    # error downloading file
+    pass
+
+
+
+def cur_org():
+    return organization
+
+def report_date():
+    return today
+
+def finalize_defaults(org=None, date=None):
+    org = org if org else cur_org()
+    date = date if date else report_date()
+    return org, date
+
+def get_data_for_org(org=None, date=None):
+    org, date = finalize_defaults(org, date)
+    srcname = f"{date}-{org}.db.obj.json"
+    data = get_data_from_file(srcname)
+    return data
+
+
+@functools.lru_cache()
```

Comment:
I don't think you will want to cache this, since it's reading from a file?

Reply:
Same goes for nearly all functions in this file.

Reply:
The file contents don't change during a run. Actually, I've removed this cache, and cache only after the expensive operation of converting the file contents to JSON. The file access routine is only called once per file.